### PR TITLE
Kc update

### PR DESCRIPTION
**Pull request by kev1n80**
+ Switched from using English words as values for the filter input tags to using numbers
+ Made filter input keyboard accessible (using tabs and enter key)
+ Added a see all button for the filter inputs that makes all of the filters checked and shows listing without any filters applied
Screenshot 2020-08-07 at 6 33 29 PM

### CODE REVIEW

File: src/main/webapp/scripts/listing-preview.js [hunk 1]
```diff
@@ -21,21 +24,15 @@ import { toggleDisplay } from './miscellaneous.js';
...
-      toggleDisplay(listingDisplay, listingId);
-    }
-  });
+
+  const toggleListingDisplay = () => {
+    toggleDisplay(listingDisplay, listingId);
+  }
+
+  // // Make section listing keyboard accessible 
+  // //     (which includes tabs and enter keys)
```
Comment (zrideaux) [inline src/main/webapp/scripts/listing-preview.js:33]: Should these comments have 4 slashes?

File: src/main/webapp/scripts/search.js [hunk 2]
```diff
@@ -23,6 +30,60 @@ export default function displayListings() {
   getListings(containerElement, '', 'search-listings', queryString);
 }
 
+/**
+ * Add on click and when submit key is pressed to checkbox/radio inputs and to 
+ *     see all span element.
+ */
+function addOnclickToInputs() {
+  // // Add onclick and onsubmit function to see all span element.
```
Comment (zrideaux) [inline src/main/webapp/scripts/search.js:38]: Should this have 4 slashes as well?

**Comment by kev1n80** [comment]
> LGTM, nice tidying up the HTML! Curious if you think alphabetical CSS is preferable to grouping things into different sections (not to say search.css had perfect groups prior)?

Thank you! At first I grouped CSS based on relevance, but as I went back to try to find specific CSS classes I found it easier if the CSS classes were in alphabetical order.

File: src/main/webapp/scripts/listing-preview.js (hunk 1 above)
Comment (kev1n80) [reply src/main/webapp/scripts/listing-preview.js:33]: Nope, nice catch!

File: src/main/webapp/scripts/search.js (hunk 2 above)
Comment (kev1n80) [reply src/main/webapp/scripts/search.js:38]: No hahaha, good catch.

**Pull request merged**